querybook/webapp/lib/query-result/helper.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * checks if a cell of results table is empty
+ * only applies to query results in querybook
+ *
+ * @param val cell from query execution result
+ * @returns boolean
+ */
+export function isCellValNull(val: any): boolean {
+    return val === 'null' || val == null;
```

Comment:
is it possible to be empty string "" or uppercase "NULL"?

Reply:
no the serialization is done by Querybook, so we know exactly what NULL would be converted to